src/audio_sdl.cpp

```diff
@@ -70,9 +74,27 @@ SdlAudio::SdlAudio() :
 		return;
 	}
 
+#ifdef GEKKO
+	// Wii's DSP works at 32kHz natively
+	const int frequency = 32000;
+#elif defined(EMSCRIPTEN)
+	// Get preferred sample rate from Browser (-> OS)
+	const int frequency = EM_ASM_INT_V({
+		var context;
+		try {
+			context = new AudioContext();
+		} catch (e) {
+			context = new webkitAudioContext();
+		}
+		return context.sampleRate;
```

Comment:
Isn't this a memory leak?

Reply:
This is JavaScript code so will be garbage collected.